source/index.txt

```diff
@@ -10,6 +12,105 @@ MongoDB Shell
    :depth: 1
    :class: singlecol
 
+.. include:: /includes/admonitions/fact-mdb-shell-beta.rst
+
+The |mdb-shell| is an interactive JavaScript shell interface to MongoDB,
+which provides a powerful interface for users to test queries and
+operations directly with the database. The |mdb-shell| also provides a
+fully functional JavaScript environment for use with MongoDB.
```

Comment:
[suggestion] I think this could be simplified some, maybe something like:

The |mdb-shell| is an interactive JavaScript environment for interfacing with a MongoDB deployment. You can use the shell to test queries and other operations directly with a database.

Reply:
Agree. This was largely borrowed from the existing copy in the manual, but it could be simplified.